references/detection/transforms.py

```diff
@@ -282,3 +282,47 @@ def forward(
                 image = F.to_pil_image(image)
 
         return image, target
+
+
+class ScaleJitter(nn.Module):
+    """Randomly resizes the image and its bounding boxes  within a specified ratio range.
+    The class implements the Scale Jitter augmentation as described in the paper
+    `"Simple Copy-Paste is a Strong Data Augmentation Method for Instance Segmentation" <https://arxiv.org/abs/2012.07177>`_.
+
+    Args:
+        scale_range (tuple of ints): scaling factor interval, e.g (a, b), then scale is randomly sampled from the
+            range a <= scale <= b.
+        interpolation (InterpolationMode): Desired interpolation enum defined by
+            :class:`torchvision.transforms.InterpolationMode`. Default is ``InterpolationMode.BILINEAR``.
+    """
+
+    def __init__(
+        self,
+        scale_range: Tuple[float, float] = (0.1, 2.0),
```

Comment:
This is fine as is, but this means user will need to apply a `Resize(target_size)` before applying `ScaleJitter`.

This is fine and probably more composable than having this function resize internally as well, although it could be slightly inefficient as we will be performing two interpolations of the image.

Reply:
Do you mean "apply resize AFTER"?

Reply:
I spoke offline with @fmassa and we decided to refactor this and add an extra `target_size` parameter in this transform to make it stand-alone.

Reply:
More handling of the bbox case was needed to support the `target_size`. This is fixed at #5446